GSA-TTS/FAC
# Header Updates (Links)

# Header Updates: Links reorganized and pointed at FAC.gov

Issue: https://github.com/GSA-TTS/FAC/issues/2169

## Changes:
1. Add `static_site_url` to `context_processors.py`. This returns `STATIC_SITE_URL` in context dict format.
2. Add `config.context_processors.static_site_url` to `settings.TEMPLATES.OPTIONS.context_processors`, making `STATIC_SITE_URL` a global template variable
3. Change the `STATIC_SITE_URL` in settings to be fac.gov
    * In the future we can make this conditional, and use different static site URLS in different cases. Like, say, a different static link for a beta test on a preview environment. 
5. Use the newly global `STATIC_SITE_URL` in the primary nav component for links. 

New changes! 
1. Updated the primary nav to match closer with the static site. It doesn't include the site search, but links are good to go, and support links are wrapped into a dropdown. 

New new changes!
Removed the beta banner.

This doesn't break any tests, because no tests cover these links. Cypress may, eventually, but for now we must manually check them. 

## How to test:
1. Switch to this branch and start normally.
2. Verify that the header matches with the static site (screenshot below). 
3. Verify that the links in the primary nav now go to FAC.gov instead of a federalist branch. 
4. Ensure these links persist on several pages. 

## Screenshots:
The app (this change):
![image](https://github.com/GSA-TTS/FAC/assets/91098850/a9771272-1e12-4f3f-8ceb-471623b46346)

The static site:
![image](https://github.com/GSA-TTS/FAC/assets/91098850/07cd043a-1579-43c9-83b7-8a634eb4e0ab)

## PR checklist: submitters


## PR checklist: reviewers

        
The larger the PR, the stricter we should be about these points.
